### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Available Software_ Screen Reader: NVDA is not reading the “More” link.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3360]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #NVDA; #Java tooling for IntelliJ/Eclipse; #C+E-H
 
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
 
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**Screen Reader:** NVDA
**MAS Violated:** MAS1.3.1

 **NOTE: Same issue is observed in “Install Details” screen as well.**

**Repro Steps:** 
1) Start Screen Reader and 0pen Java EE - Eclipse application.
2) Tab till Help-->Install New software.
3)  Enter http://dl.microsoft.com/eclipse/ in 'Work with' text box and press enter.
4)Tab to all controls and verify how screen reader is reading.
5)Verify whether NVDA  is reading “More” link or not.

**Actual Result:** 
NVDA is not reading the “More” link.

 **Expected Result:** 
NVDA should  read the “More” link.

 **User Impact:** 
Screen Reader can not get the information about “More” link if screen reader is not reading it.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
![Bug_3360_NVDA issue](https://user-images.githubusercontent.com/38523250/62208808-7f6fda00-b3b5-11e9-93f6-6a3bfd93dab8.png)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The install ui is controlled by eclipse, which is out control of our plugin, here is the related issue in Eclipse https://bugs.eclipse.org/bugs/show_bug.cgi?id=550649, NVDA can't read an Eclipse Link again with tab.

Here is a gif of install AWS Eclipse Toolkits, we could see the same behavior of this issue.
![3360](https://user-images.githubusercontent.com/12445236/64939875-fb09e580-d894-11e9-9e1f-4c84299747ec.gif)

**Comment by Flanker32** [comment]
- Importance of experience: Little, it's not a common behavior to open more link while install new software
- Quality of experience: Little, screen reader could read the links for the first time, and it could read it again when hover on it.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749794
